interface/patient_file/upload_dialog.php

```diff
@@ -45,6 +45,9 @@
     if (!file_exists($imagedir  )) mkdir($imagedir  );
     check_file_dir_name($what);
     $filename = "$imagedir/$what.jpg";
+    $targetFile = basename($filename);
+    $imageFileType = pathinfo($targetFile,PATHINFO_EXTENSION);
+    
```

Comment:
Avoid adding white spaces here and there. Limit that please

Reply:
Wrong in my opinion.  Separation of distinct code elements makes it easier to read.  Limiting is fine, indentions and line returns where appropriate is good.  That is the standard we have used for a pretty long time.  Having all the code look like a single block is pretty rough for old people eyes.  You would know nothing about that yet, so just take my word.  :)

Reply:
yep right